src/models/user.ts

```diff
@@ -3,21 +3,28 @@ import { Document } from 'dynamoose/dist/Document';
 import Semester, { SemesterInterface } from './semester';
 
 export const userSchema = new dynamoose.Schema(
-    {
-        id: String,
-        semesters: {
-            type: Array,
-            schema: Semester,
-        },
-    },
-    {
-        timestamps: true,
-    }
+	{
+		id: {
+			type: String,
+			hashKey: true,
+		},
+		semesters: {
+			type: Set,
+			schema: [Semester],
+		},
+		owner: {
```

Comment:
No owner field for the user, the id should be the `cognito:username` (ie, userId) and therefore no owner should be on the schema